stl/inc/any

```diff
@@ -10,7 +10,7 @@
 #if _STL_COMPILER_PREPROCESSOR
 
 #if !_HAS_CXX17
-#pragma message("The contents of <any> are available only with C++17 or later.")
+#pragma message(_WARNING_MESSAGE("STL4049", "The contents of <any> are available only with C++17 or later."))

```

Comment:
Why have a different number for every header? They are all the same error, just reported by different headers. This will cause a lot of headache when adding new headers.

Reply:
I use different number because it's used before.

See:

https://github.com/microsoft/STL/blob/bd7adb4a932725f60ba096580c415616486ab64c/stl/inc/yvals_core.h#L566

https://github.com/microsoft/STL/blob/bd7adb4a932725f60ba096580c415616486ab64c/stl/inc/yvals_core.h#L572

https://github.com/microsoft/STL/blob/bd7adb4a932725f60ba096580c415616486ab64c/stl/inc/yvals_core.h#L576

Same error, 3 different numbers...

But I agree that one number for the error will be easy when adding a new header.